Bottom: 84042bab651ba3409c38614a38204df99af34ef6
Top:    c797f95060a14a6baa8d7871d5f03aefaa694536
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 12:42:47 +0000

Refresh of more-play-with-quad-edge-data-STEP

---

diff --git a/src/hid/common/face3d.h b/src/hid/common/face3d.h
index d6ef68d..695c1a5 100644
--- a/src/hid/common/face3d.h
+++ b/src/hid/common/face3d.h
@@ -13,7 +13,6 @@ typedef struct {
 
   /* XXX: STEP specific - breaks encapsulation */
   int plane_identifier;
-  bool plane_orientation_reversed;
   int face_identifier;
   int face_bound_identifier;
 } face3d;
